geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionStats.java

```diff
@@ -425,8 +435,16 @@
                 false),
 
             f.createLongGauge("localMaxMemory",
-                "local max memory in bytes for this region on this member", "bytes")
-
+                "local max memory in bytes for this region on this member", "bytes"),
+            f.createLongCounter("clearLocalDuration",
```

Comment:
I think clearLocalDuration and clearTotalDuration should no longer exist. If I'm right then you want to delete these two lines

Reply:
Done

Reply:
you are right.